cmd/ctr/commands/client.go

```diff
@@ -50,17 +54,34 @@ func AppContext(context *cli.Context) (gocontext.Context, gocontext.CancelFunc)
 		log.L.Debugf("Using SOURCE_DATE_EPOCH: %v", tm)
 		ctx = epoch.WithSourceDateEpoch(ctx, tm)
 	}
+
 	return ctx, cancel
 }
 
 // NewClient returns a new containerd client
 func NewClient(context *cli.Context, opts ...containerd.ClientOpt) (*containerd.Client, gocontext.Context, gocontext.CancelFunc, error) {
+	// containerd's gRPC connections don't have credentials regardless of OpenTelemetry.
+	dialOpts := []grpc.DialOption{
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
+	}
```

Comment:
Do we need to add this in if containerd already does? https://github.com/containerd/containerd/blob/8b15917123d1a4e42f1013b37142fb0fb5c8bec3/client.go#L124

Reply:
Yeah. Adding otelgrpc's interceptors through dialOpts below resets the default dialOpts.

Reply:
And apparently that's the cause of the Windows failure below!

Reply:
Yea, I couldn't think of anything else that could've been it. What is actually going on then (if you've debugged far enough)?

Reply:
Not that far. Maybe the lack of `grpc.WithContextDialer(dialer.ContextDialer)`?